Firebase/Core/FIRErrors.m

```diff
@@ -19,3 +19,4 @@
 NSString *const kFirebaseCoreErrorDomain = @"com.firebase.core";
 NSString *const kFirebasePerfErrorDomain = @"com.firebase.perf";
 NSString *const kFirebaseStorageErrorDomain = @"com.firebase.storage";
+NSString *const kFirebaseSegmentationErrorDomain = @"com.firebase.segmentation";
```

Comment:
Delete.  FIRErrors is deprecated and nothing new should be added.

Reply:
Done.